src/JBrowse/View/FileDialog.js

```diff
@@ -140,7 +140,12 @@ return declare( null, {
         else {
             on( localFilesControl.uploader, 'click', function() {
                 var dialog = electronRequire('electron').remote.dialog;
-                var ret = dialog.showOpenDialog({ properties: [ 'openFile','multiSelections' ]});
+                var ret;
+                if(process.env.SPECTRON) {
+                    ret = ["docs/tutorial/data_files/volvox.fa"];
```

Comment:
Note that is is difficult to "mock" the system dialog apparently so this seems like a sort of workaround for testing...